.github/workflows/lint.yml

```diff
@@ -28,8 +28,8 @@ jobs:
         with:
           exclude: "./vendor/..."
           name: "Revive"
-      - name: Install golint
-        run: go get golang.org/x/lint/golint
+      #- name: Install golint
```

Comment:
Let's just directly remove this. Other than that, let's merge this and address the violations in another PR.

Reply:
On a second thought, I'll also remove this when addressing the violations ;)